### PR TITLE
Adding Example 3, in case of complex string including other characters

### LINKED ISSUE #4153: [Edit] Strings: Adding codebyte for the entry `.toLowerCase()`
### Type of Edit (select all that apply)

Add new content (definitions, codeblocks, etc.)

### Description (optional)

[PLEASE DO NOT WORK ON THIS ISSUE UNTIL YOU HAVE BEEN ASSIGNED] We would like an edit to the `toLowerCase.md` entry. The term entry lives in docs/content/javascript/concepts/strings/terms/toLowerCase/toLowerCase.md.

Please refer to the [term entry templates](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

Please:
1) Add a codebyte which shows how we can use `.toLowerCase()`.
2) You can be creative with the codebyte snippet but make sure you don't deviate a lot from the entry.

### Codebyte Example

image

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by dcldavid**
### Description

Added Example 3 to the `toLowerCase()` method featuring how the method can be used in more complex string use cases.
Closes #4153 

### Type of Change

- Updating the documentation

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR

### CODE REVIEW

**Comment by dcldavid** [comment]
First ever pull request! Apologies in advance for errors!

File: content/javascript/concepts/strings/terms/toLowerCase/toLowerCase.md [hunk 1]
```diff
@@ -41,3 +41,16 @@ console.log(message)
 
 console.log(message.toLowerCase());
 ```
+
+## Example 3
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/strings/terms/toLowerCase/toLowerCase.md:45]:
```suggestion
## Codebyte Example
```

File: content/javascript/concepts/strings/terms/toLowerCase/toLowerCase.md [hunk 2]
```diff
@@ -41,3 +41,16 @@ console.log(message)
...
+
+The `.toLowerCase()` method will convert upper case alphabet characters to lower case in a string with other character types such as digits and special characters.
+
+```codebyte/js
+const string = 'What a COMPLEX string !1@2#3'
+
+const newString = string.toLowerCase()
+console.log(newString)
+// Output: what a complex string !1@2#3
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/strings/terms/toLowerCase/toLowerCase.md:54]:
```suggestion
```
No need to show the output here as a comment the readers can compile the code and get the output as usual.

File: content/javascript/concepts/strings/terms/toLowerCase/toLowerCase.md [hunk 3]
```diff
@@ -41,3 +41,16 @@ console.log(message)
 
 console.log(message.toLowerCase());
 ```
+
+## Example 3
+
+The `.toLowerCase()` method will convert upper case alphabet characters to lower case in a string with other character types such as digits and special characters.
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/strings/terms/toLowerCase/toLowerCase.md:47]:
```suggestion
Here `.toLowerCase()` method will convert upper-case alphabet characters to lower-case in a string with other character types such as digits and special characters:
```

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
I have requested few changes from my side for you @dcldavid 🚀

**Comment by dcldavid** [comment]
Thanks @dakshdeepHERE I have committed all of the changes.

**Review by dakshdeepHERE — APPROVED** [review]
LGTM 🚀

**Review by ishg-153 — APPROVED** [review]
@dcldavid I made a small change but everything else looks good. Ready to merge!

**Pull request merged**